Parser/tokenizer.c

```diff
@@ -1844,6 +1850,10 @@ tok_get(struct tok_state *tok, char **p_start, char **p_end)
     /* Line continuation */
     if (c == '\\') {
         c = tok_nextc(tok);
+        if ( tok->async_def == 2){
```

Comment:
Coding style, please write: "if (...) {" (fix spacing)

Reply:
Done in 7518077